mock-api/vpc.ts

```diff
@@ -45,6 +45,97 @@ export const vpc2: Json<Vpc> = {
 
 export const vpcs: Json<Vpc[]> = [vpc, vpc2]
 
+export const defaultRouter: Json<VpcRouter> = {
+  id: 'fc59fb4d-baad-44a8-b152-9a3c27ae8aa1',
+  name: 'mock-system-router',
+  description: 'a fake router',
+  time_created: new Date(2024, 0, 1).toISOString(),
+  time_modified: new Date(2024, 0, 2).toISOString(),
+  vpc_id: vpc.id,
+  kind: 'system',
+}
+
+export const customRouter: Json<VpcRouter> = {
+  id: '7ffc1613-8492-42f1-894b-9ef5c9ba2507',
+  name: 'mock-custom-router',
+  description: 'a fake custom router',
+  time_created: new Date(2024, 1, 1).toISOString(),
+  time_modified: new Date(2024, 1, 2).toISOString(),
+  vpc_id: vpc.id,
+  kind: 'custom',
+}
+
+export const vpcRouters: Json<VpcRouter[]> = [defaultRouter, customRouter]
+
+const routeBase = {
+  time_created: '2024-07-11T17:46:21.161086Z',
+  time_modified: '2024-07-11T17:46:21.161086Z',
+  vpc_router_id: defaultRouter.id,
+}
+
+export const routerRoutes: Json<Array<RouterRoute>> = [
+  {
+    ...routeBase,
+    id: '51e50342-790f-4efb-8518-10bf01279514',
+    name: 'default',
+    description: "VPC Subnet route for 'default'",
+    kind: 'vpc_subnet',
+    target: {
+      type: 'subnet',
+      value: 'default',
+    },
+    destination: {
+      type: 'subnet',
+      value: 'default',
+    },
+  },
+  {
+    ...routeBase,
+    id: '4c98cd3b-37be-4754-954f-ca960f7a5c3f',
+    name: 'default-v4',
+    description: 'The default route of a vpc',
+    kind: 'default',
+    target: {
+      type: 'internet_gateway',
+      value: 'outbound',
+    },
+    destination: {
+      type: 'ip_net',
+      value: '0.0.0.0/0',
```

Comment:
This is a real nitpick, but this would be more realistic-looking as `192.168.1.0/24` and the v6 as `2001:db8:abcd:12::/64`.